zokrates_cli/examples/book/rng_tutorial/test.sh

```diff
@@ -4,7 +4,7 @@ set -e
 bin=$1; stdlib=$2
 
 function zokrates() {
-  ZOKRATES_STDLIB=$stdlib $bin $*
+  ZOKRATES_STDLIB=$stdlib $bin "$@"
```

Comment:
just curious: what does this do?

Reply:
This was interesting, to give context this function is used as a wrapper to have cleaner commands in the book (we reference the lines in this script). I had an issue where if you would pass a string for one of the commands eg. `-e "1 2"` it would pass it down as `-e 1 2` which is wrong because now `2` is outside the string and is passed as a separate parameter which would result in errors like `invalid parameter "2"`. By using `"$@"` these groups are preserved because the expansion logic is a bit different